api/envoy/extensions/filters/http/tap/v3/tap.proto

```diff
@@ -26,4 +26,15 @@ message Tap {
   // Common configuration for the HTTP tap filter.
   common.tap.v3.CommonExtensionConfig common_config = 1
       [(validate.rules).message = {required: true}];
+
+  // Specific configuration except common config for the HTTP tap filter.
+  OutputConfig output_config = 2;
+}
+
+//  Specific configuration to control trace output
```

Comment:
Per style guide, please add a period ('.') at the end of a sentence in comments.